compiler/rustc_mir/src/borrow_check/diagnostics/conflict_errors.rs

```diff
@@ -678,6 +678,15 @@ impl<'cx, 'tcx> MirBorrowckCtxt<'cx, 'tcx> {
             None,
         );
 
+        match explanation {
+            BorrowExplanation::UsedLater(LaterUseKind::Call, _call_span) => {
+                err.help(
+                    "try adding a local storing the argument and then using the local in the call",
+                );
```

Comment:
It would be nice to have a structured suggestion, but I'm guessing that would be really complicated. Unless it's not as hard as I think to implement a structured suggestion for this, I think let's focus in this PR on emitting the help message only when it makes sense. Then maybe I can do a followup PR if you have ideas for structured suggestions :)